ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh

```diff
@@ -97,4 +100,8 @@ cd "$(mktemp -d)"
 
 git clone --branch main "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
 make ci/prepare/e2e-branch
-make ci/test/e2e
+
+pac_host=https://$(kubectl -n openshift-pipelines get route pipelines-as-code-controller -o jsonpath='{.spec.host}')
+export pac_host
```

Comment:
this won't work, since `openshift-pipelines` is not yet created on the test cluster at this stage.

this part also needs to be done after bootstrapping the cluster, i.e. after boostrap is complete
1. [here](https://github.com/redhat-appstudio/e2e-tests/blob/88bd3e467ec9acc3711a0b28da69a1c9e6bf1b30/magefiles/magefile.go#L112) for local dev setup
2. [here](https://github.com/redhat-appstudio/e2e-tests/blob/88bd3e467ec9acc3711a0b28da69a1c9e6bf1b30/magefiles/magefile.go#L224) for CI

Reply:
thanks, I removed pac_host variable. fixed.